defaults/main.yml

```diff
@@ -1,5 +1,5 @@
 ---
-theo_agent_version: "0.10.1"
```

Comment:
this should be in a separate commit, i.e "bump theo_agent_version to 0.11.0"

Reply:
Done